test/e2e/benchmark/benchmark.go

```diff
@@ -63,7 +65,33 @@ func (b *BenchmarkTest) SetupNodes() error {
 		testnet.WithTimeoutPropose(b.manifest.TimeoutPropose),
 		testnet.WithTimeoutCommit(b.manifest.TimeoutCommit),
 		testnet.WithPrometheus(b.manifest.Prometheus),
+		testnet.WithLocalTracing(b.manifest.LocalTracingType),
 	))
+
+	if b.manifest.PushTrace {
+		log.Println("reading trace push config")
+		if pushConfig, err := trace.GetPushConfigFromEnv(); err == nil {
+			log.Print("Setting up trace push config")
+			for _, node := range b.Nodes() {
+				if err = node.Instance.SetEnvironmentVariable(trace.
+					PushBucketName, pushConfig.BucketName); err != nil {
```

Comment:
uber nit, it seems odd to introduce a newline here

```suggestion
				if err = node.Instance.SetEnvironmentVariable(trace.PushBucketName, pushConfig.BucketName); err != nil {
```

Reply:
Agree, fixed https://github.com/celestiaorg/celestia-app/pull/3514/commits/2df5bf382a3660f3ffe8c30a63d4aafabc4ba2f6